hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneChaosCluster.java

```diff
@@ -203,7 +203,7 @@ void initializeConfiguration() throws IOException {
           1, TimeUnit.SECONDS);
       conf.setTimeDuration(HddsConfigKeys.HDDS_HEARTBEAT_INTERVAL, 1,
           TimeUnit.SECONDS);
-      conf.setInt(OzoneConfigKeys.OZONE_CONTAINER_CACHE_SIZE, 8);
+      conf.setInt(OzoneConfigKeys.OZONE_CONTAINER_CACHE_SIZE, 2);
```

Comment:
One question: Any reason for changing this value?

Reply:
Thanks for the review @bharatviswa504. This value is being reduced otherwise we run into too many open files exception.